### DISCUSSION TITLE
Support for HTTP/2

### DISCUSSION THREAD

**Discussion by Saphyel** [Kludex/uvicorn#2070] [Ideas]
I was wondering if there's any plans to support for HTTP/2 ? if not what is the reason behind ? 

I can understand the lack of HTTP/3 because is fairly new but HTTP/2 has been around almost 9 years and arguably gives some performance improvements.

**Comment by omerXfaruq** [comment]
With this discussion I would like to suggest thinking about a roadmap for HTTP/3 as well. It is also possible to skip HTTP/2 and directly jump to HTTP/3.
[Here](https://github.com/aiortc/aioquic.git) is a library for HTTP/3

PS. HTTP/3 might go into another discussion.
